src/app/modules/members/store/members.actions.ts

```diff
@@ -0,0 +1,41 @@
+import { ApolloError } from '@apollo/client/errors';
+import { createAction, props } from '@ngrx/store';
+import { getDictionaryDynamicProperty } from 'src/app/graphql/types/getDictionaryDynamicProperty';
+import { Member } from 'src/app/models/member.model';
+
+export const getGender = createAction(
```

Comment:
Rename to getGenderValues or Items

Reply:
Already passed test on QA, so will rename to getGenderDictionaryItems in next PR